pkg/tenant/db/projects_test.go

```diff
@@ -262,7 +303,19 @@ func (s *dbTestSuite) TestUpdateProject() {
 	require.NoError(db.UpdateProject(ctx, project), "could not update project")
 	require.Equal(project.Modified, project.Created, "expected created timestamp to be updated")
 
+	// Should fail if project ID is missing
+	project.ID = ulid.ULID{}
+	require.ErrorIs(db.UpdateProject(ctx, project), db.ErrMissingID, "expected error for missing project ID")
+
+	// Should fail if project is invalid
+	project.ID = ulid.MustParse("01GKKYAWC4PA72YC53RVXAEC67")
+	project.Name = ""
+	require.ErrorIs(db.UpdateProject(ctx, project), db.ErrMissingProjectName, "expected error for invalid project")
```

Comment:
```suggestion
	require.ErrorIs(db.UpdateProject(ctx, project), db.ErrMissingProjectName, "expected missing project name error")
```

Reply:
This mirrors an error message `TestCreateTenantProject` and may be a little more descriptive.